docs/0.4.0-alpha/getting_started/installation.md

```diff
@@ -15,11 +15,11 @@ Make sure that the operating system meets the following prerequisites
 
 - **System-wide**
 ```bash
-bash <(curl -s "https://raw.githubusercontent.com/amber-lang/amber/master/setup/install.sh")
+bash <(curl -sL "https://github.com/amber-lang/amber/releases/download/0.4.0-alpha/install.sh")
```

Comment:
You have `--` following the `bash` command elsewhere, I'm guessing so that `--user` is passed to the downloaded install script, rather than `bash` itself.  Consider adding here for consistency:
```
bash -- <(curl -sL "https://github.com/amber-lang/amber/releases/download/0.4.0-alpha/install.sh")
```

Reply:
did you tested the command? because we had issues in the past and I found that this command works.

Reply:
I did; `curl` successfully downloads the script for me:
```
$ curl -sL "https://github.com/amber-lang/amber/releases/download/0.4.0-alpha/install.sh" | head
#!/usr/bin/env bash
# Written in [Amber](https://amber-lang.com/)
# version: 0.4.0-alpha
# date: 2024-12-21 20:03:44

file_exists__33_v0() {
    local path=$1
     [ -f "${path}" ] ;
    __AS=$?;
if [ $__AS != 0 ]; then
```

Reply:
the problem is not just downloading but executing, I did the same mistake and I had to change again the command.